### DISCUSSION TITLE
Remove unused repos

### DISCUSSION THREAD

**Discussion by Conengmo** [python-visualization/folium#1970] [Collaborators]
- can we remove the https://github.com/python-visualization/folium_contrib repo? I don't think it's relevant anymore...
- I'm pretty sure we can also remove https://github.com/python-visualization/Leaflet.awesome-markers. We created that fork since somebody offered to make improvements on it. But that never happened, and we never started using this fork. See https://github.com/python-visualization/folium/issues/923#issuecomment-429231796.

**Comment by ocefpaf** [comment]
Yep. Go for it!

**Comment by Conengmo** [comment]
Done! Thanks!
